asab/storage/__init__.py

```diff
@@ -26,5 +27,12 @@ def __init__(self, app):
 			from .elasticsearch import StorageService
 			self.Service = StorageService(app, "asab.StorageService")
 
+		# new configuration format
+		elif sttype is None:
+			# [elasticsearch]
+			if asab.Config.has_section('elasticsearch'):
+				from .elasticsearch import StorageService
+				self.Service = StorageService(app, "asab.StorageService", config_section_name='elasticsearch')
+
 		else:
-			L.error("Unknown asab:storage type '{}'".format(sttype))
+			L.error("Unknown asab:storage type '{}' or unsupported configuration format".format(sttype))
```

Comment:
This makes the log message confusing for me. When I get an error, it should be quite clear how should I repair it. In this case, `StorageService` does not know this storage provider. The log about wrong elastic configuration should be elsewhere IMO.